examples/blogs/delta1.1/merge_code_example.py

```diff
@@ -0,0 +1,27 @@
+pyspark --packages io.delta:delta-core_2.12:1.1.0 --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" --conf "spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog"
+
+#1. Merge operation with the feature disabled
+
+from delta.tables import *
+#define the delta table on which the merge operation is being performed
```

Comment:
it would  be good to improve the python coding style a bit. like 
```suggestion
# Define the delta table on which the merge operation is being performed
```